msi/GitProduct.wxs

```diff
@@ -264,6 +264,8 @@
   </Fragment>
 
   <Fragment>
+    <SetProperty Id="ExecConfigureSslCAInfo" Value='"[#BashExe]" -c "git config -f \"/[#GfwGitConfig]\" http.sslCAinfo \"/[#ca_bundle.crt]\""' Sequence="execute" Before="ExecConfigureSslCAInfo" />
```

Comment:
On Fri, 13 May 2016, Craig E. Shea wrote:

> > @@ -264,6 +264,8 @@
> >    </Fragment>
> >  
> >    <Fragment>
> > +    <SetProperty Id="ExecConfigureSslCAInfo" Value='"[#BashExe]" -c "git config -f \"/[#GfwGitConfig]\" http.sslCAinfo \"/[#ca_bundle.crt]\""' Sequence="execute" Before="ExecConfigureSslCAInfo" />
> 
> Based on your comments here and on Issue 531, if I understand correctly, on Windows-based systems, Git needs to "learn" about another level of configurtaion settings, i.e. {`%ProgramData\Git\config`, system, global, local}. This comes into play when, for example, `libgit2` is being used on a Windows machine _and_ Git for Windows is installed (or possibly other utilities that make use of git and git configuration settings) and they wish to share certain git configuration settings. I may not have the details correct here, but the point is, the MSYS2 _system_ configuration file for Git should remain to be located at `INSTALLDIR\mingw??\etc\gitconfig`, and all settings selected during installation of Git for Windows should be made in this file, including this `http.sslCAInfo` setting.
> 
> Is this correct?

This is exactly what I wanted to say. :smile: